modules/client/examples/04-decoders/08-update-plugin-settings-token.ts

```diff
@@ -11,7 +11,7 @@ import { contextParams } from "../00-client/00-context";
 const context: Context = new Context(contextParams);
 // Create a plugin context from the simple context
 const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
-const clientAddressList = new ClientErc20(contextPlugin);
+const clientAddressList = new ClientAddressList(contextPlugin);
```

Comment:
this should be TokenVotingClient, no?

Reply:
Either the variable name or the class name looks wrong